protozfits/__init__.py

```diff
@@ -72,7 +62,7 @@ def __exit__(self, type, value, tb):
         self.close()
 
     def close(self):
-        File.instances -= 1
+        pass
 
     def __del__(self):
```

Comment:
I think it would be nice to have context management. I.e. that there would be something like available:
```python
with File('blabla.fits.fz') as f:
```

To do this one needs to create `__enter__()` and `__exit__()` functions in `File`

I would say same for `MultiZfitsFile`. 
I found this pretty usefull : http://book.pythontips.com/en/latest/context_managers.html

Reply:
very good point ...

Reply:
Ah .. I just realize:
https://github.com/cta-sst-1m/protozfitsreader/blob/6901c5449761183d64d8d0aa9f946010818bd075/protozfits/__init__.py#L60-L64
`File` already is a context manager